models/icepack/readme.rst

```diff
@@ -0,0 +1,81 @@
+.. _icepack:
+
+Icepack
+=======
+
+Overview
+--------
+
+DART interface modules for Icepack, the column physics of the sea ice model CICE (`https://github.com/cice-consortium/Icepack <https://github.com/cice-consortium/Icepack>`_). Icepack is maintained by the CICE Consortium.
+
+A large portion of the physics in sea ice models can be described in a vertical column, without reference to neighboring grid cells. This code includes several options for simulating sea ice thermodynamics, mechanical redistribution (ridging) and associated area and thickness changes. In addition, the model supports a number of tracers, including thickness, enthalpy, ice age, first-year ice area, deformed ice area and volume, melt ponds, and biogeochemistry.
+
+More information about the model and instructions on how to run Icepack can be found in the `Icepack documentation <https://cice-consortium-icepack.readthedocs.io/en/main/index.html>`_.
+
+This model is run as a separate executable from DART, and this means that you must use scripts to alternate the model and DART program execution and allow for the progression of the assimilation through multiple time windows. These scripts will be provided by DART, but they are currently still in progress. 
+
+The assimilation process can be easily executed within a single assimilation window, however. There is a test case available in ``/glade/work/masmith/test_cases/icepack_test`` that contains the necessary input files to run filter, the main program in DART that performs the assimilation. There is a README file in this directory to give more details on the specifics of the test case. If you do not have access to the NSF NCAR Derecho Supercomputer, the reach out to the DAReS team at ``masmith@ucar.edu`` and we will provide you with the test case.
+
+The steps to run this example are as follows:
+
+1.  | ``cd /DART/models/icepack/work``
+    | Navigate to the work directory for Icepack
+
+2.  | ``./quickbuild.sh`` (or ``quickbuild.sh nompi`` if you are not building with mpi)
+    | Builds all DART executables 
+
+3.  | ``cp -r /glade/work/masmith/test_cases/icepack_test ..``
+    | Copy the test directory from the directory stated above to your Icepack directory
+
+4.  | ``cd ../icepack_test``
+    | Navigate to the test directory
+
+5.  | In ``work/input.nml``, set ``perturb_from_single_instance = .true.`` in the
+      ``&filter_nml``
+    | This setting causes filter to perturb a single restart file to generate an
+      ensemble
+
+6.  | ``./filter``
+    | Runs the assimilation program, resulting in four main output files:
+    |    ``analysis_mean.nc`` and ``analysis_sd`` - the mean and standard deviation of the state of all ensemble members after the assimilation
+    |    ``obs_seq.final`` - the ensemble members' estimate of the observations.
+    |    ``dart_log.out`` - detailed log file for the execution of filter
```

Comment:
No need for this section.  There is plenty of dart documentation on how to run filter. 

```suggestion
```

As an aside do you have a test case with an actual ensemble? Perturbing might be funky for icepack because (I think) it has bounded qualities (thickness).

Reply:
Test case that uses an acutal ensemble of 30 members is in /glade/derecho/scratch/masmith/DART_latest/DART/models/icepack/tests/icepack_test